relay-dynamic-config/src/project.rs

```diff
@@ -90,6 +93,35 @@ impl ProjectConfig {
         metrics::convert_conditional_tagging(self);
         defaults::add_span_metrics(self);
     }
+
+    /// Combines the measurementsconfig from global config and project config.
```

Comment:
nit: could we link to the structures in the docs here?

Reply:
oh yeah, good call!